daemon/algod/api/client/restClient.go

```diff
@@ -332,24 +335,42 @@ type accountInformationParams struct {
 
 // TransactionsByAddr returns all transactions for a PK [addr] in the [first,
 // last] rounds range.
+// Deprecated: This function is only used in internal tests (stateproofs_test.go, restClient_test.go)
 func (client RestClient) TransactionsByAddr(addr string, first, last, max uint64) (response v1.TransactionList, err error) {
 	err = client.get(&response, fmt.Sprintf("/v1/account/%s/transactions", addr), transactionsByAddrParams{first, last, max})
 	return
 }
 
 // PendingTransactionsByAddr returns all the pending transactions for a PK [addr].
+// Deprecated: Use v2 API
 func (client RestClient) PendingTransactionsByAddr(addr string, max uint64) (response v1.PendingTransactions, err error) {
 	err = client.get(&response, fmt.Sprintf("/v1/account/%s/transactions/pending", addr), pendingTransactionsByAddrParams{max})
 	return
 }
```

Comment:
Is the plan to leave this one endpoint active on the v1 API, or will it be removed separately?

Reply:
It will be removed in a separate PR along with any tests in `restClient_test.go` that use it. Although the endpoint has no v2 equivalents, it looked okay to be replaced by `/v2/account/%s/transactions/pending` in internal tests.

Reply:
Thanks for confirming, I saw all the other deprecation notices after leaving this comment. Good plan :+1: